service-api/module/Application/src/Helpers/ServiceAvailabilityHelper.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Helpers;
+
+use Application\Model\Entity\CaseData;
+use Application\Fixtures\DataWriteHandler;
+use Application\Sirius\EventSender;
+use Psr\Log\LoggerInterface;
+
+class ServiceAvailabilityHelper
+{
+    /**
+     * @psalm-suppress PropertyNotSetInConstructor
+     */
+    protected array $availableServices;
```

Comment:
Recommend initialising this 
```suggestion
    protected array $availableServices = [];
```